test/routes/home_route_test.dart

```diff
@@ -69,4 +72,75 @@ void main() {
       expect(ContentControl.instance.state.allSongs.songs, [songWith()]);
     });
   });
+
+  testWidgets('app shows exit confirmation toast', (WidgetTester tester) async {
```

Comment:
Can we rather have:
* one test that verifies that for `true` - both that the app shows toast and doesn't call pop with setting set to true, and only allows to pop after a certain duration
* and the other test that checks that with `false` we immediately are popped

I like such structure more, because it tests the entire feature flow, as if we are looking at it from a user's perspective, and it could be more clearely formalized, if we were writing a test checklist, for example

Reply:
Done